app/build.gradle

```diff
@@ -65,6 +86,10 @@ android {
         buildConfigField "String", "NPS_SURVEY_URL", '"https://dev-wallet-net-promoter-score.radixdlt.com/"'
         buildConfigField "String", "REFINER_FORM_UUID", '"281622a0-dc6b-11ee-8fd1-23c96056fbd2"'
         resValue "string", "app_name", "Radix Wallet"
+
+        def properties = secretProperties()
+        buildConfigField "String", "APPS_FLYER_KEY", buildConfigProperty(properties, 'appsFlyerDevKey')
```

Comment:
Maybe rename this too `appsFlyerDevKey` => `appsFlyerKey`

Reply:
Changing the key name here involves changing it also in CI configuration files an I'm not sure it's worth it, since the 'dev' in the context of the AF key is valid, this is what the key is called on their platform and it's not related to the environment.